modules/phase_field/include/postprocessors/FauxGrainTracker.h

```diff
@@ -61,6 +50,7 @@ class FauxGrainTracker : public FeatureFloodCount, public GrainTrackerInterface
   const int _tracking_step;
 
   std::vector<std::pair<unsigned int, unsigned int> > _faux_data;
+  std::vector<unsigned int> _faux_data_2;
```

Comment:
Don't worry about that. If you guys like this, I"ll immediately submit
another PR deprecating the existing API and will rename these variables.
Right now there are two data structures with the same information in them.

On Mon, Jul 25, 2016 at 9:00 AM Daniel Schwen <notifications@github.com>
wrote:

> In modules/phase_field/include/postprocessors/FauxGrainTracker.h
> <https://github.com/idaholab/moose/pull/7375#discussion_r72079726>:
>
> > @@ -61,6 +50,7 @@ class FauxGrainTracker : public FeatureFloodCount, public GrainTrackerInterface
> >    const int _tracking_step;
> >
> >    std::vector<std::pair<unsigned int, unsigned int> > _faux_data;
> > +  std::vector<unsigned int> _faux_data_2;
>
> I'd really like to see some doc strings here. The new variable name with
> the _2 appended to it is pretty nondescript.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/idaholab/moose/pull/7375/files/771a20b36cff9efce184c3edc1f69921dbe2a023#r72079726>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AC5XICiYBNzqzvgaZ2J_TbJRLarxeMnlks5qZM-BgaJpZM4JRNvk>
> .
>